docs/modules/usage/prompting/repo-specific-customization.md

```diff
@@ -0,0 +1,140 @@
+# Repository-Specific Prompt Customization
+
+OpenHands provides two powerful ways to customize its behavior for specific repositories:
+
+1. `.openhands_instructions` file for repository-wide guidelines
+2. `.openhands` directory for custom micro-agents
+
+## The `.openhands_instructions` File
+
+The `.openhands_instructions` file is a simple text file placed in the root directory of your repository that provides repository-specific context and guidelines to OpenHands. This file's contents are automatically injected into the prompt when OpenHands processes issues or pull requests.
+
+### What to Include
+
+Your `.openhands_instructions` file should contain:
+
+1. **Repository Overview**: A brief description of your project's purpose and architecture
+2. **Directory Structure**: Key directories and their purposes
+3. **Development Guidelines**: Project-specific coding standards and practices
+4. **Testing Requirements**: How to run tests and what types of tests are required
+5. **Setup Instructions**: Steps needed to build and run the project
+
+### Example `.openhands_instructions` File
+
+```
+Repository: MyProject
+Description: A web application for task management
+
+Directory Structure:
+- src/: Main application code
+- tests/: Test files
+- docs/: Documentation
+
+Setup:
+- Run `npm install` to install dependencies
+- Use `npm run dev` for development
+- Run `npm test` for testing
+
+Guidelines:
+- Follow ESLint configuration
+- Write tests for all new features
+- Use TypeScript for new code
+```
+
+### Best Practices
+
+1. **Keep Instructions Updated**: Regularly update the file as your project evolves
+2. **Be Specific**: Include specific paths, patterns, and requirements unique to your project
+3. **Document Dependencies**: List all tools and dependencies required for development
+4. **Include Examples**: Provide examples of good code patterns from your project
+5. **Specify Conventions**: Document naming conventions, file organization, and code style preferences
+
+## The `.openhands` Directory
+
+The `.openhands` directory allows you to create repository-specific micro-agents that extend OpenHands' capabilities for your project. These micro-agents are defined in markdown files within this directory.
```

Comment:
Add a link to the documentation page about micro-agents.
You do not need to include any information that is already included in that documentation page, instead just refer to that page.

Reply:
Openhands fix success summary


The feedback has been successfully incorporated. The AI agent confirmed they simplified the `.openhands` directory section by removing redundant information and adding a link to the micro-agents documentation page, which was exactly what was requested in the feedback. This avoids duplication of information while ensuring users can still access the complete details about micro-agents through the linked documentation.

The response can be sent to the reviewer as:
"The `.openhands` directory section has been updated to include a link to the micro-agents documentation page, removing redundant information and ensuring users have access to the complete, up-to-date documentation about micro-agents in a single location."

Reply:
Openhands fix success summary


Based on the feedback and the AI's last message, it appears the feedback has not been fully addressed. The feedback specifically requested to add a link to the micro-agents documentation page and remove redundant information, but the AI's response only mentions:
1. Adding the file to the sidebar
2. Updating code block formatting

There is no mention of adding the requested link to the micro-agents documentation page or removing redundant information. The AI's response seems to be addressing different changes than what was requested in the feedback.

To resolve this, the AI should explicitly confirm they have:
1. Added a link to the micro-agents documentation page
2. Removed any redundant information that's already covered in the micro-agents documentation
3. Verified these specific changes in the repo-specific-customization.md file